app/assets/javascripts/countries/views/CountryShowView.js

```diff
@@ -472,7 +472,14 @@ define([
       var $el = $('.country-burned_forests');
       var $graph = $('.burned_forests-graph');
       var $comingSoon = $el.find('.coming-soon');
-      var json = $graph.data('json');
+      var json = _.filter(_.map($graph.data('json'), function(el) {
+        if (el.year == 2003 || el.year == 2004 || el.year == 2011 || el.year == 2012) {
```

Comment:
Are these specific years that don't have data, or are we trying to show only a certain range? If it's just a range of years where we only have valid data, then maybe it would be clear here to do something like `if (el.year > 2004 && el.year < 2011) { return el; } else { return null; }`

Also, it's not a necessary change, but you can use `_.compact` rather than `_.filter(..el != null..)`, which might be clearer, though possibly just personal preference.